src/container/Movies.js

```diff
@@ -1,14 +1,53 @@
 import React,{Component} from "react"
 import axios from 'axios'
+import Moviescss from "./Moviescss.css"
 import Movie_list from "../components/Movie_list"
```

Comment:
In javascript, we follow camelCase naming convention for . So, try to use variables like movieList instead of movie_list.